cypress/integration/CTS/SearchResultsPageTests/index.js

```diff
@@ -0,0 +1,189 @@
+    });
+});
+
+And('{string} button is displayed', (expectedButton) => {
+    cy.get('.cts-results-header button').should('be.visible').and('contain.text', expectedButton);
+});
+
+
+When('user clicks on {string} button', (searchCriteriaButton) => {
+    cy.get('button.cts-accordion__button').contains(searchCriteriaButton).click();
+});
+
+And('search criteria table displays the following', (dataTable) => {
+    cy.get('tbody > tr').should('have.length', dataTable.hashes().length);
+    //index assures correct order
+    let index = 0;
+    for (const { Category, Selection } of dataTable.hashes()) {
+        cy.get('tbody tr th').eq(index).should('have.text', Category);
+        cy.get('tbody tr td').eq(index).should('have.text', Selection);
+        index++;
+    }
+});
+
+And('{string} link has a href {string}', (linkText, linkHref) => {
+    cy.get('a').contains(linkText).should('have.attr', 'href').and('eq', linkHref);
+});
+
+And('{string} pager is displayed', (pagerPosition) => {
+    cy.get(`div.results-page__control.--${pagerPosition} .results-page__pager`).should('be.visible');
+});
+
+And('result list is displayed', () => {
+    cy.get('div.results-list').should('be.visible');
+});
+
+And('each result item is a link', () => {
+    cy.document().then((doc) => {
+        const titleLocator = doc.querySelectorAll('.results-list-item__title');
+        for (let i = 0; i < titleLocator.length - 1; i++) {
+            cy.get(titleLocator[i]).should('be.visible');
+            cy.get(titleLocator[i]).find('a').should('have.attr', 'href');
+        }
+    });
+});
+
+And('there are {int} delighters present', (count) => {
+    cy.get('.cts-delighter-container div[class^="delighter"]:visible').should('have.length', count);
+});
+
+And('the following delighters are displayed', (dataTable) => {
+    for (const { delighter, href, title, text } of dataTable.hashes()) {
+        cy.get(`div[class="delighter ${delighter}"]:visible`).as('delighter');
+        cy.get('@delighter').find('a').first().should('have.attr', 'href', href);
+        cy.get('@delighter').find('h4').first().should('have.text', title);
+        cy.get('@delighter').find('p').first().should('have.text', text);
+    }
+});
+
+And('chat pop up is displayed', () => {
+    cy.wait(30000);
+    cy.get('#ProactiveLiveHelpForCTSPrompt').should('be.visible');
+});
+
+And('chat pop up title is {string}', (chatTitle) => {
+    cy.get('#ProactiveLiveHelpForCTSPrompt').find('h2').should('contain', chatTitle);
+});
+
+And('{string} button has a link {string}', (chatNowButton, link) => {
+    cy.get('#proactive-chat-form').should('have.attr', 'action', link);
+    cy.get('#chat-button').should('have.text', chatNowButton);
+});
+
+
+When('user clicks on Chat Now button', () => {
+    cy.get('form').then((form$) => {
+        form$.on('submit', (e) => {
+            e.preventDefault();
+            cy.get('button#chat-button').click();
+        });
+    });
```

Comment:
well, this step is good when you are testing event that is fired on the click( analytics, for instance) - for you test scenario it is obsolete, since you are checking if the chat will open by looking for a _blank target on the button

So how about this - manually test chat pop up for analytics event and add it to the separate scenario ( using this step) . As for _Scenario:  Search results page displays the Chat Pop Up_ - remove this step from feature file all together and rename the one below to 
`And clicking on chat now button will open a chat in a new tab`